Makefile.am

```diff
@@ -347,8 +347,11 @@ COMMITTED_DIST = \
 # Build up the distribution using $COMMITTED_DIST and include node_modules and bower licenses
 # also automatically update minimum base dependency in RPM spec file
 dist-hook::
-	( cd $(srcdir); git ls-tree HEAD --name-only -r $(COMMITTED_DIST) || (echo $(COMMITTED_DIST) | tr ' ' '\n' ) ) | \
-		tar -C $(srcdir) -cf - -T - | tar -C $(distdir) -xf -
+	( if [ -d $(srcdir)/.git ]; then \
+		git -C $(srcdir) ls-tree HEAD --name-only -r $(COMMITTED_DIST) | tar -C $(srcdir) -cf - -T -; \
+	else \
+		tar -C $(srcdir) -cf - $(COMMITTED_DIST); \
```

Comment:
That would also include any built files, backup files etc. in pkg/, test/, and tools/. So our tarball in `distcheck` would move from "containing too little" to "containing too much", and thus actually become less reliable. I do understand that it's annoying to enumerate every single file (and I don't want to do that either), but I wonder if making it "differently broken" is worth the effort.

Perhaps this should rather use `ls-tree --full-tree`? This then would catch a random git in the parent dir outside of the the build tree, so it's also not perfect, but at least it would build a correct dist tar during distcheck.